src/commcare_cloud/commands/migrations/plays/assert_couch_files.yml

```diff
@@ -7,21 +7,26 @@
         path: "/opt/data/couchdb2/{{item}}"
       loop: "{{ files_by_node[inventory_hostname].shards }}"
       register: shards
+      when: inventory_hostname in files_by_node
 
     - name: stat views
       stat:
         path: "/opt/data/couchdb2/{{item}}"
       loop: "{{files_by_node[inventory_hostname].views }}"
       register: views
+      when: inventory_hostname in files_by_node
 
     - name: assert shards exist
       assert:
         that: "{{ item.stat.exists }}"
         fail_msg: "Shard file missing: {{ item.item }}"
       loop: "{{ shards.results }}"
+      when: inventory_hostname in files_by_node
 
     - name: assert views exist
       assert:
         that: "{{ item.stat.exists and item.stat.isdir }}"
         fail_msg: "View missing: {{ item.item }}"
       loop: "{{ views.results }}"
+      when: inventory_hostname in files_by_node
```

Comment:
is this because you moved all data to new nodes but the old nodes were still in the inventory?

Reply:
Yeah exactly.